assets/js/modules/analytics/components/dashboard/LegacyDashboardAcquisitionPieChart.js

```diff
@@ -90,11 +90,11 @@ const LegacyDashboardAcquisitionPieChart = ( { data, source } ) => {
 	return (
 		<div className="googlesitekit-chart googlesitekit-chart--pie">
 			<GoogleChart
+				chartType="PieChart"
```

Comment:
Another observation for this chart is that the new version cuts highlight curves:

![Screenshot from 2021-04-23 17-21-22](https://user-images.githubusercontent.com/873049/115886120-4a44ce80-a459-11eb-9b0b-11f71984c317.png)

Reply:
Odd, I'm not able to recreate that:

![2021-04-29 13 02 27](https://user-images.githubusercontent.com/90871/116547776-3d0d6100-a8eb-11eb-89e1-e8f7c69d2d13.gif)

Reply:
I can't reproduce that bug locally; which browser are you using to cause that bug?

<img width="644" alt="Screenshot 2021-04-29 at 15 41 25" src="https://user-images.githubusercontent.com/90871/116569509-5f11de00-a901-11eb-9528-2d92ed431598.png">

Reply:
Ah, I do see it now. Weird that it only happens at large viewports. I'll see what's causing it 👍🏻

Reply:
Ah, I caught it, we had a CSS fix for it in the old charts code that I used for the new one. Nice catch 👍🏻